index_configuration_tool/tests/test_main.py

```diff
@@ -20,13 +20,16 @@
 # Utility method to create a test plugin config
 def create_plugin_config(host_list: list[str],
                          user: Optional[str] = None,
-                         password: Optional[str] = None) -> dict:
+                         password: Optional[str] = None,
+                         disable_auth: Optional[bool] = None) -> dict:
     config = dict()
     config["hosts"] = host_list
     if user:
         config["username"] = user
     if password:
         config["password"] = password
+    if disable_auth is not None:
+        config["disable_authentication"] = disable_auth
```

Comment:
I was gonna comment if this should be `is not False` but then I got the idea that you're only checking if it has a value at all first, then assigning that value (no matter what it is) if it exists to the `disable_authentication` config attribute (I might be using the wrong term here). Anyways, I somehow really like this if condition :)